Objects/interface/UnpackedPFCand.h

```diff
@@ -13,28 +13,6 @@ namespace panda {
 
   class UnpackedPFCand : public ParticleM {
   public:
-    enum PType {
```

Comment:
Seems like we're losing a few things here. The enum is fine (duplicated in `PFCand`), but we don't want access to `q` and `pdgId`?